ToArcGISEarth/ToArcGISEarth/ToArcGISEarth.csproj

```diff
@@ -95,30 +95,22 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <AddInContent Include="Config.daml" />
-    <AddInContent Include="Images\AddInDesktop16.png" />
-    <AddInContent Include="Images\AddInDesktop32.png" />
-    <AddInContent Include="DarkImages\AddInDesktop16.png" />
-    <AddInContent Include="DarkImages\AddInDesktop32.png" />
+    <AddInContent Include="Config.daml">
+      <SubType>Designer</SubType>
+    </AddInContent>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="EarthNamedpipeAPIUtils.cs" />
-    <Compile Include="IEarthNamedpipeService.cs" />
+    <Compile Include="AddLayerCheckBox.cs" />
+    <Compile Include="AutomationAPI\EarthNamedpipeAPIUtils.cs" />
+    <Compile Include="AutomationAPI\IEarthNamedpipeService.cs" />
+    <Compile Include="ClearArcGISEarthWorkspaceButton.cs" />
     <Compile Include="Module1.cs" />
+    <Compile Include="AutomationAPI\ProcessUtils.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="SyncToArcGISEarthButton.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <AddInContent Include="Images\GenericButtonBlue16.png" />
-  </ItemGroup>
-  <ItemGroup>
-    <AddInContent Include="Images\GenericButtonBlue32.png" />
-  </ItemGroup>
-  <ItemGroup>
-    <AddInContent Include="DarkImages\GenericButtonBlue16.png" />
-  </ItemGroup>
-  <ItemGroup>
-    <AddInContent Include="DarkImages\GenericButtonBlue32.png" />
+    <Compile Include="RemoveLayerCheckBox.cs" />
+    <Compile Include="SaveImageFromArcGISEarthButton.cs" />
+    <Compile Include="SyncToArcGISEarthCheckBox.cs" />
+    <Compile Include="ConnectToArcGISEarthButton.cs" />
   </ItemGroup>
   <ItemGroup>
     <AddInContent Include="Images\ArcGISEarth16.png" />
```

Comment:
Not sure we need change this name to ArcGISEarth18.png or just ArcGISEarth.png

Reply:
No, ArcGISEarth16.png just represents image size, there also have ArcGISEarth32.png.